x-pack/plugins/upgrade_assistant/server/saved_object_types/reindex_operation_saved_object_type.ts

```diff
@@ -15,13 +15,25 @@ export const reindexOperationSavedObjectType: SavedObjectsType = {
   mappings: {
     properties: {
       reindexTaskId: {
-        type: 'keyword',
+        type: 'text',
```

Comment:
Note that I made these other changes to regain parity with how the dynamic mapping was originally mapping these fields.

Reply:
This matches the default mappings for a dynamic field detected as being text, but is there any value in maintaining parity? With this mapping two fields will be added, text and keyword. It seems unlikely that we would do a match query on the keyword, so maybe having only a text field (with `index: false`) would add less indexing overhead without removing any validation by ES.

Reply:
Thanks for raising this and for suggesting a solution! I decided to maintain parity to minimize the risk of introducing another regression as much as possible. I have no idea if there's any value beyond that for maintaining parity (haven't looked into it yet). I agree that many of these fields probably don't need to be indexed as keyword (or possibly at all). https://github.com/elastic/kibana/issues/64547 tracks the work of updating these mappings to minimize the number of fields we're indexing and I'll keep your suggestion in mind when I begin working on it.